using_images/xpaas_images/eap.adoc

```diff
@@ -1,119 +1,110 @@
 
-----
-$ /opt/eap/bin/jboss-cli.sh
-----
+== xPaaS JBoss EAP Version Compatibility Matrix
 
-[WARNING]
-Any configuration changes made using the JBoss EAP Management CLI on a running container will be lost when the container restarts.
+See the xPaaS section of the https://access.redhat.com/articles/2176281[OpenShift and Atomic Platform Tested Integrations page] for details about OpenShift EAP image version compatibility.
 
-link:#Making-Configuration-Changes-EAP[Making configuration changes to the
-JBoss EAP instance inside the JBoss EAP xPaaS image] is different from the process you may be used to for a regular release of JBoss EAP.
+== Set Up the xPaaS JBoss EAP Image
 
-=== Unsupported Configurations
+The following is a list of prerequisites for using the xPaaS JBoss EAP images:
 
-The following is a list of unsupported configurations specific to the JBoss EAP xPaaS image:
+. **Acquire Red Hat Subscriptions** - Ensure that you have the relevant subscriptions for OpenShift as well as a subscription for xPaaS Middleware.
 
-* Using MySQL in a scaled environment with XA distributed transactions is not recommended. For applications that support both scaling and XA distributed transactions, PostgreSQL is recommended instead.
-// This is based on https://issues.jboss.org/browse/CLOUD-56
+. **Install OpenShift** - Before using the xPaaS JBoss EAP images, you must have an OpenShift environment installed and configured:
 
-ifdef::openshift-enterprise[]
-== Using the JBoss EAP xPaaS Image Streams and Application Templates
+.. The link:../install_config/install/quick_install.html method allows you to install OpenShift using an interactive CLI utility.
 
-The Red Hat xPaaS middleware images were
-link:../../install_config/imagestreams_templates.html[automatically created during the installation]
-of OpenShift along with the other default image streams and templates.
-endif::[]
+.. The link:../install_config/install/advanced_install.html method allows you to install OpenShift using a reference configuration. This method is best suited for production environments.
 
-[[Making-Configuration-Changes-EAP]]
-== Running and Configuring the JBoss EAP xPaaS Image
+.. Install and Deploy Docker Registry - Install the Docker Registry and then ensure that the Docker Registry is deployed to locally manage images as follows:
++
+----
+$ oadm registry --config=/etc/origin/master/admin.kubeconfig --credentials=/etc/origin/master/openshift-registry.kubeconfig
+----
++
+For further information, see link:../install_config/install/docker_registry.html
 
-You can make changes to the JBoss EAP configuration in the xPaaS image using either the S2I templates, or by using a modified JBoss EAP xPaaS image.
+. *Deploy a Router* - Use the instructions at the link:../install_config/install/deploy_router.html page for this step.
```

Comment:
You used double asterisks above for `**Install OpenShift**` so why only single asterisks here?